### DISCUSSION TITLE
Definitive test for bricking

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#377] [Q&A]
Is there a simple definitive test to tell if a Slingbox is bricked ? e.g. different lights displayed or ping to slingbox IP address wont work

Would be good to know that the box is ok before trying to diagnose any other issues.

**Comment by sp222222** [comment]
pretty simple really - put slingbox on open, unblocked network and in about 10secs it will be used as a doorstop........  the thing will ping and the lights will look normal too. 
If this is the AV, tuner, SOLO or PRO they will brick unless the IP they use is rule block on the router it sits on.  IF you own a m1,m2 350, 500 or ProHD, merry XMAS, you got lucky not using port blocking rules (yet).  Most of us on here have been burned already by bricks, don't join the crowd on this one.

**Comment by RandomMan719** [comment]
> Is there a simple definitive test to tell if a Slingbox is bricked ? e.g. different lights displayed or ping to slingbox IP address wont work
> 
> Would be good to know that the box is ok before trying to diagnose any other issues.

Don't connect the SlingBox to the internet!
What model of Slingbox do you have? 
I am pretty sure newer models, have not been bricked, yet.  You just can't configure them, currently.
If you have an older model, you can open SlingPlayer 1.5 and select the SlingBox in the Directory and go to the information tab and it will show an error, if it is bricked.
![image](https://user-images.githubusercontent.com/116921195/204877595-0944ce9b-a07d-44b6-b3e4-7849ad58a3ba.jpeg)
